CODE_OF_CONDUCT.md

```diff
@@ -12,13 +12,13 @@ body size, race, ethnicity, age, religion, or nationality.
 
 Examples of unacceptable behavior by participants include:
 
-* The use of sexualized language or imagery
-* Personal attacks
-* Trolling or insulting/derogatory comments
-* Public or private harassment
-* Publishing other's private information, such as physical or electronic
+- The use of sexualized language or imagery
+- Personal attacks
+- Trolling or insulting/derogatory comments
+- Public or private harassment
+- Publishing other's private information, such as physical or electronic
```

Comment:
There's some unrelated changes from running prettier. I can configure prettier to only run on JavaScript files and revert the changes if you prefer @mpalmer.

Reply:
Cool, I will add it to the .prettierignore file and revert the changes :) prettier also formats any other YAML, JSON and Markdown files by default. Let me know if you want me to disable it - I personally don't mind it but appreciate it could be annoying especially given it will fail the CI.